iset.mm

```diff
@@ -27453,6 +27453,28 @@ practical reasons (to avoid having to prove sethood of ` A ` in every use
       IAJBCKPQALABCMN $.
   $}
 
+  $( Demonstrate by witnesses that two classes lack a subclass relation.
+     (Contributed by Stefan O'Rear, 5-Feb-2015.) $)
+  nelss $p |- ( ( A e. B /\ -. A e. C ) -> -. B C_ C ) $=
+    ( wcel wss ssel com12 con3dimp ) ABDZBCEZACDZJIKBCAFGH $.
+
+  ${
+    ssrexf.1 $e |- F/_ x A $.
+    ssrexf.2 $e |- F/_ x B $.
+    $( restricted existential quantification follows from a subclass
```

Comment:
```suggestion
    $( Restricted existential quantification follows from a subclass
```
(this is also the case in set.mm, optional)

Reply:
Yeah, it wasn't new, but sure, I'll change it.